Makefile

```diff
@@ -82,11 +82,6 @@ $(KORE_SUBMODULE)/make.timestamp:
 	touch ${KORE_SUBMODULE}/make.timestamp
 
 ocaml-deps: .build/local/lib/pkgconfig/libsecp256k1.pc
-	opam init --quiet --no-setup
```

Comment:
I think this should still call k-configure-opam, right?

Reply:
No, I'm assuming the user has gone through the K setup. I'll add instructions to the README.

I figure the Makefile shouldn't be setting up things which happen outside the repo, and this one has caused strife for people.

Reply:
@dwightguth I've added instructions to the README about setting up K's OCaml dependencies.